xcube/api/gen/gen.py

```diff
@@ -54,7 +54,7 @@ def gen_cube(input_files: Sequence[str] = None, input_processor: str = None, inp
     Generate a data cube from one or more input files.
 
     :param sort_mode:
-    :param input_files: The input files.
+    :param input_files: The input files or a txt-file containing the paths of the input files.
```

Comment:
Just "The input files.", because the logic to read from text files is somewhrere else.